src/components/NotificationRow.tsx

```diff
@@ -80,11 +87,11 @@ export const NotificationRow: React.FC<IProps> = ({
         <NotificationIcon size={18} aria-label={notification.subject.type} />
       </div>
 
-      <div className="flex-1 overflow-hidden">
+      <div className="flex-1 overflow-hidden" onClick={() => pressTitle()}>
         <div
           className="mb-1 text-sm whitespace-nowrap overflow-ellipsis overflow-hidden cursor-pointer"
           role="main"
-          onClick={() => pressTitle()}
+          // onClick={() => pressTitle()}
```

Comment:
```suggestion
```
No need to keep this, it is pretty self-explanatory if we want to rollback!

Reply:
Ooops, left that there by accident.  Great catch